packages/server/src/migrations/functions/pluginCount.ts

```diff
@@ -0,0 +1,17 @@
+import { tenancy, logging } from "@budibase/backend-core"
+import { plugins } from "@budibase/pro"
+import env from "../../environment"
+
+export const run = async () => {
+  // only a self hosted op
+  if (!env.SELF_HOSTED) {
```

Comment:
Is it possible there are inaccurate plugin counts in the cloud also? Or why would we not run there?

Reply:
If we did want to retain this check we should probably do it outside the migration, in the case statement for building migrations. 

That means the record of the migration run will never be recorded, which is better in case we do want to then run this in cloud.

Reply:
no plugins are currently unavailable in the cloud due to another bug.

Reply:
Cool np. Still the same point about not actually running the migration,we probably shouldn't populate the record unless the migration actually runs.

Reply:
Is it possible the quota got incorrect set for any tenants while plugins were available in cloud?

Reply:
plugins have never actually been available in the cloud - there was an error meaning that it was impossible to add plugins - which will be fixed as part of this release.